temp/AKSeImageTemplateWindows.json

```diff
@@ -0,0 +1,187 @@
+        "managed_identity_name": {
+            "type": "string"
+        },
+        "vnet_resource_group_name": {
+          "type": "string"
+        },
+        "vnet_name": {
+            "type": "string"
+        },
+        "subnet_name": {
+            "type": "string"
+        },
+        "image_gallery_name": {
+            "type": "string"
+        },
+        "image_version": {
+            "type": "string"
+        },
+        "vhd_url": {
+            "type": "string",
+            "defaultValue": "https://akseashvhds.blob.core.windows.net"
+        },
+        "vhd_blob_container_name": {
+            "type": "string",
+            "defaultValue": "test"
+        },
+        "vhd_name": {
+            "type": "string"
+        },
+        "vhd_run_output_name": {
+            "type": "string"
+        },
+        "github_branch_name": {
+            "type": "string",
+            "defaultValue": "master"
+        },
+        "location": {
+            "type": "string"
+        },
+        "vm_size": {
+            "type": "string"
+        },
+        "build_number": {
+            "type": "string"
+        },
+        "build_id": {
+            "type": "string"
+        },
+        "build_repo": {
+            "type": "string"
+        },
+        "git_version": {
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "name": "[parameters('template_name')]",
+            "type": "Microsoft.VirtualMachineImages/imageTemplates",
+            "apiVersion": "2022-02-14",
+            "location": "[parameters('location')]",
+            "dependsOn": [],
+            "tags": {
+                "userIdentity": "enabled"
+            },
+            "identity": {
+                "type": "UserAssigned",
+                "userAssignedIdentities": {
+                    "[concat('/subscriptions/',parameters('subscription_id'),'/resourcegroups/',parameters('vhd_resource_group_name'),'/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('managed_identity_name'))]": {}
+                }
+            },
+            "properties": {
+                "buildTimeoutInMinutes" : 180,
+                "vmProfile": 
+                    {
+                        "vmSize": "[parameters('vm_size')]",
+                        "osDiskSizeGB": 127,
+                        "vnetConfig": {
+                            "subnetId": "[concat('/subscriptions/',parameters('subscription_id'),'/resourcegroups/',parameters('vnet_resource_group_name'),'/providers/Microsoft.Network/virtualNetworks/',parameters('vnet_name'),'/subnets/',parameters('subnet_name'))]"
+                        }
+                    },
+                "source": {
+                    "type": "PlatformImage",
+                    "publisher": "MicrosoftWindowsServer",
+                    "offer": "WindowsServer",
+                    "sku": "2019-Datacenter-Core-smalldisk",
+                    "version": "17763.5329.231230"

```

Comment:
version should probably be a pipeline variable

Reply:
In the current pipeline, these variables are also from hardcoded values (https://github.com/Azure/aks-engine-azurestack/blob/master/vhd/packer/windows-2019-vars.json), so maybe okay?